lms/static/js/learner_dashboard/views/course_entitlement_view.js

```diff
@@ -0,0 +1,399 @@
+                 },
+
+                 enrollSuccess: function(data) {
+                     /*
+                     Update external elements on the course card to represent the now available course session.
+
+                     1) Show the change session toggle button.
+                     2) Add the new session's dates to the date field on the main course card.
+                     3) Hide the 'View Course' button to the course card.
+                     */
+                     var successIconEl = '<span class="fa fa-check" aria-hidden="true"></span>';
+
+                     // Update the model with the new session Id;
+                     this.entitlementModel.set({currentSessionId: this.currentSessionSelection});
+
+                     // Allow user to change session
+                     this.$triggerOpenBtn.removeClass('hidden');
+
+                     // Display a success indicator
+                     HtmlUtils.setHtml(this.$dateDisplayField,
+                         HtmlUtils.joinHtml(
+                             HtmlUtils.HTML(successIconEl),
+                             this.getAvailableSessionWithId(data.course_run_id).session_dates
+                         )
+                     );
+
+                     // Ensure the view course button links to new session home page and place focus there
+                     this.$enterCourseBtn
+                         .attr('href', this.formatCourseHomeUrl(data.course_run_id))
+                         .removeClass('hidden')
+                         .focus();
+                     this.toggleSessionSelectionPanel();
+                 },
+
+                 unenrollSuccess: function() {
+                     /*
+                     Update external elements on the course card to represent the unenrolled state.
+
+                     1) Hide the change session button and the date field.
+                     2) Hide the 'View Course' button.
+                     3) Remove the messages associated with the enrolled state.
+                     4) Remove the link from the course card image and title.
+                     */
+
+                     // Update the model with the new session Id;
+                     this.entitlementModel.set({currentSessionId: this.currentSessionSelection});
+
+                     // Reset the card contents to the unenrolled state
+                     this.$triggerOpenBtn.addClass('hidden');
+                     this.$enterCourseBtn.addClass('hidden');
+                     this.$courseCardMessages.remove();
+                     this.$('.enroll-btn-initial').focus();
+                     HtmlUtils.setHtml(
+                        this.$dateDisplayField,
+                         HtmlUtils.joinHtml(
+                             HtmlUtils.HTML('<span class="icon fa fa-warning" aria-hidden="true"></span>'),
+                             HtmlUtils.HTML(gettext('You must select a session below to access the course.'))
```

Comment:
Sorry for the nittiness @HarryRein :-) Please also strike the word "below"

Reply:
Removed